client/Modals/DisplaySidebar.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
```

Comment:
Should add a //TODO comment for incomplete code so that I know it's not intended to be functional.

Reply:
Ok! thank you!!